sdcm/cluster_k8s/eks.py

```diff
@@ -645,8 +667,8 @@ def add_nodes(self,
                                       rack=rack,
                                       enable_auto_bootstrap=enable_auto_bootstrap)
         for node in new_nodes:
-            self.k8s_cluster.set_security_groups(node.ec2_host)
-            self.k8s_cluster.set_tags(node.ec2_host.id)
+            self.k8s_cluster.set_security_groups(node.ec2_instance_id)
+            self.k8s_cluster.set_tags(node.ec2_instance_id)
```

Comment:
why not setting it when provisioning instances e.g. in `get_cluster_k8s_eks`?

Reply:
Because we terminate nodes (which causes auto-recreation) in nemesis and do upgrades.
In these cases we get new nodes which do not have tags and sgs be applied.

Reply:
there no way to apply those on the nodegroup ?

Reply:
> there no way to apply those on the nodegroup ?

https://github.com/scylladb/scylla-cluster-tests/blob/38b7b3e04abfc33029d87b5e6b13792967ee9460/sdcm/cluster_k8s/eks.py#L457
https://github.com/scylladb/scylla-cluster-tests/blob/38b7b3e04abfc33029d87b5e6b13792967ee9460/sdcm/cluster_k8s/eks.py#L470

Reply:
Seems like we might consider switch to Launch Templates, so we can pass both Tags and SGs in it.

It's shame that they don't do it out of the box...

Reply:
@vponomaryov can you open a task to checkout if how complex is gonna be to use Launch Templates for the node group ?

Reply:
@fruch https://github.com/scylladb/qa-tasks/issues/1155